homeworks/gaalistvantamas/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,38 @@
+"""
+Author: Gaál István Tamás
+Task: Homework-2 / 1 
+"""
+
+DAYS_OF_A_YEAR = 365.25
+
+
+# Ask for a name and convert
+name = str(input("Please enter your name!\n"))
+
+formated_name = name.strip().title()
```

Comment:
nem hiba, de mivel a name változót később sehol nem használod, így 1 soron is letudhatod:
`formatted_name = input("Please enter your name!\n").strip().title()`